db/migrate/20150119102846_create_firms_in_person_advice_methods.rb

```diff
@@ -0,0 +1,14 @@
+class CreateFirmsInPersonAdviceMethods < ActiveRecord::Migration
+  def change
+    create_table :firms_in_person_advice_methods, id: false do |t|
+      t.belongs_to :firm
+      t.belongs_to :in_person_advice_method
+    end
+
+    add_index :firms_in_person_advice_methods, :firm_id,
```

Comment:
No need for a custom named index here, only for the `in_person_advice_method` field due to it's length.